### PR TITLE
Bump to 0.8.14, update conduit contracts, refactor test fixtures

### PR DESCRIPTION
**Pull request by d1ll0n**

### CODE REVIEW

File: contracts/lib/TokenTransferrerConstants.sol [hunk 1]
```diff
@@ -33,6 +33,7 @@ pragma solidity >=0.8.7;
  *      codebase but have been left in for readability.
  */
 
+uint256 constant AlmostOneWord = 0x19;
```
Comment (leonardoalt) [inline contracts/lib/TokenTransferrerConstants.sol:36]: Huh shouldn't this be 31? 0x1f instead

Comment (gpersoon) [reply contracts/lib/TokenTransferrerConstants.sol:36]: how about `OneWord -1` ?

Comment (leonardoalt) [reply contracts/lib/TokenTransferrerConstants.sol:36]: better!

Comment (leonardoalt) [reply contracts/lib/TokenTransferrerConstants.sol:36]: Actually doesn't compile:
```solidity
TypeError: Only direct number constants and references to such constants are supported by inline assembly.
   --> contracts/lib/TokenTransferrer.sol:662:51:
    |
662 |                             add(returndatasize(), AlmostOneWord),
    |                                                   ^^^^^^^^^^^^^
```
So I kept it as 0x1f in the PR
https://github.com/ProjectOpenSea/seaport/pull/388

File: contracts/Seaport.sol [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.13;
+pragma solidity >=0.8.13;
```
Comment (Thro77le) [inline contracts/Seaport.sol:2]: For caution and to avoid the unexpected, shouldn't the version be also upper bounded by `<0.9.0` or even `<=0.8.14`

**Pull request merged**